com.unity.render-pipelines.high-definition/Documentation~/Motion-Vectors.md

```diff
@@ -34,7 +34,7 @@ The options are:
 
 ## Motion vectors for transparent objects
 
-By default, HDRP does not render motion vectors for transparent Materials. This is because motion vectors from transparent GameObjects overwrite motion vectors for GameObjects behind them. For example, a window would overwrite the motion vectors for a bird flying behind it.
+By default, HDRP does not render motion vectors for transparent Materials. This is because motion vectors from transparent GameObjects overwrite motion vectors for GameObjects behind them. For example, a window would overwrite the motion vectors for a bird flying behind it. This for example will cause the bird to cause ghosting when Temporal AA is enabled as its motion vectors will be invalidated by the ones from the window.
```

Comment:
Since the bird's motion vectors are now invalid, if you used [temporal anti-aliasing](Anti-Aliasing.md#temporal-anti-aliasing-taa), the bird would produce ghosting.

Reply:
Can we keep a "For instance," or the like here? I don't like the fact that with this formulation feels like the only thing happening; this is just an example to solidify what might happen

Reply:
Sure, is continuing the example like this okay?
```
In this example, since the bird's motion vectors are now invalid, if you used [temporal anti-aliasing](Anti-Aliasing.md#temporal-anti-aliasing-taa), the bird would produce ghosting.
```

Reply:
Great!

Reply:
Done!